os/fs/driver/mtd/mtd_jedec.c

```diff
@@ -535,6 +536,70 @@ static inline void jedec_bytewrite(struct jedec_dev_s *priv, FAR const uint8_t *
 }
 #endif
 
+/************************************************************************************
+ * Name: jedec_is_erase
```

Comment:
Would you consider renaming this function to `jedec_is_erased` or `is_jedec_erased` please?